hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/keygen/TestTimestampBasedKeyGenerator.java

```diff
@@ -378,4 +382,46 @@ public void test_ExpectsMatch_MultipleInputFormats_ShortDate_OutputCustomDate()
     baseRow = genericRecordToRow(baseRecord);
     assertEquals("04/01/2020", keyGen.getPartitionPath(baseRow));
   }
+
+  @Test
+  public void test_ExpectsMatch_LogicalType_Date() throws IOException {
```

Comment:
Can we use camelcase please ? Let's follow the coding patterns used in the project.

Reply:
Had named the test based on the naming convention for existing tests in this class. Have updated names of all tests to camelCase.